azure-mgmt-subscription/dev_requirements.txt

```diff
@@ -0,0 +1,2 @@
+azure-sdk-tools
+azure-mgmt-billing
```

Comment:
sounds like a mistake, I don't see how billing could be a dependency here, maybe a failed copy/paste in the test